### PR TITLE
1.14 additional qa scripts update

### PR DESCRIPTION
**Pull request by lilachmaliniak**

### CODE REVIEW

File: deployment/get_bins [hunk 1]
```diff
@@ -38,6 +38,13 @@ LATEST_URL=http://lna-ci-slave2.kaltura.com/bin/latest/${OS}/release
 
 function download_files()
 {
+    [ -w ${LIVE_ROOT_PATH} ] && DST_DIR=${LIVE_ROOT_PATH} || DST_DIR=${VERSION}
+
+	mkdir -p "${DST_DIR}/bin"
+    mkdir -p "${DST_DIR}/liveRecorder/bin"
+
+    echo "current dir: [`pwd`]"
```
Comment (gadyaari) [inline deployment/get_bins]: Please add comments as to what you are doing for ignorants like me who don't understand all the syntax

File: deployment/upgradeLive [hunk 2]
```diff
@@ -6,30 +6,37 @@ set -e
...
     /opt/kaltura/liveController/$1/packager/bin/run_nginx.sh
+    # create symlinks to latest dir and process script
+	if [ -L "latest" ]; then
+		echo "unlink latest"
+		unlink latest
+	fi
+	if [ -L /etc/init.d/kLiveController ]; then
+		echo "unlink /etc/init.d/kLiveController"
+		unlink /etc/init.d/kLiveController
+	fi
```
Comment (gadyaari) [inline deployment/upgradeLive]: Don't understand the need to unlink the kLiveController script, as creating it again overrides it

File: deployment/get_bins (hunk 1 above)
Comment (lilachmaliniak) [inline deployment/get_bins]: done

File: deployment/upgradeLive (hunk 2 above)
Comment (lilachmaliniak) [inline deployment/upgradeLive]: you are correct. fixed

File: deployment/upgradeLive [hunk 3]
```diff
@@ -6,31 +6,38 @@ set -e
...
     ln -s /opt/kaltura/liveController/$1 /opt/kaltura/liveController/latest
-    ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController
+	if [ ! -L /etc/init.d/kLiveController ]; then
+		echo "ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController"
+		ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController
+	fi
     # start nginx (web server), liveController and liveRecorder
-    /opt/kaltura/liveController/$1/packager/bin/run_nginx.sh
     /etc/init.d/kLiveController restart
+    sleep 5
```
Comment (kalturaguy) [inline deployment/upgradeLive]: isn't there a better way knowing we can continue?

File: deployment/upgradeLive [hunk 4]
```diff
@@ -6,31 +6,38 @@ set -e
...
+    echo "### finished downloading binaries"
     # create symlinks to latest dir and process script
+	if [ -L "latest" ]; then
+		echo "unlink latest"
+		unlink latest
+	fi
     ln -s /opt/kaltura/liveController/$1 /opt/kaltura/liveController/latest
-    ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController
+	if [ ! -L /etc/init.d/kLiveController ]; then
+		echo "ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController"
```
Comment (gadyaari) [inline deployment/upgradeLive]: I would change this to: "Creating symlink to kLiveController script at - /etc/init.d/kLiveController"
(or somthing like that)

File: deployment/upgradeLive (hunk 3 above)
Comment (gadyaari) [inline deployment/upgradeLive]: not sure if this is doable, but maybe you can check the return value of the kLiveController script and only if is 0 run the nginx script?

Comment (lilachmaliniak) [inline deployment/upgradeLive]: not enough for the decision.

File: deployment/upgradeLive (hunk 4 above)
Comment (lilachmaliniak) [inline deployment/upgradeLive]: done

File: deployment/upgradeLive (hunk 3 above)
Comment (lilachmaliniak) [inline deployment/upgradeLive]: leaving it for now

**Comment by lilachmaliniak** [comment]
updated scripts for install and upgrade QA machines

**Pull request merged**